ee/se/modules/502-delivery/templates/argocd/notifications-controller/argocd-notifications-controller-deployment.yaml

```diff
@@ -71,6 +71,9 @@ spec:
             - argocd-notifications
           image: {{ include "helm_lib_module_image" (list . "argocd") }}
           imagePullPolicy: IfNotPresent
+          ports:
+          - name: argocd-notifications-port
```

Comment:
Here and everywhere:
* no need in argocd prefix (it can't be anything but argocd)
* no need in port suffix (it can't be anything but port)

Reply:
All prefixes and suffixes were deleted